scheduler/crates/infra/src/repos/schedule/postgres.rs

```diff
@@ -93,7 +75,7 @@ impl IScheduleRepo for PostgresScheduleRepo {
             schedule.id.inner_ref(),
             Json(&schedule.rules) as _,
             schedule.timezone.to_string(),
-            &metadata
+            Json(&schedule.metadata) as _,
```

Comment:
These kind of `Json` doesn't work without the `as _`. What's the secret?

Reply:
I am not really sure about the exact reason. I just noted that this is how they do it in the sqlx examples: https://github.com/launchbadge/sqlx/blob/master/examples/postgres/json/src/main.rs#L68